### PR TITLE
update doc based on postgrest/issues/1253

### PR DESCRIPTION
**Pull request by swuecho**
add a NOTE, so lazy dev do not hit the wall.

### CODE REVIEW

**Comment by swuecho** [comment]
https://github.com/PostgREST/postgrest/issues/1253

File: api.rst [hunk 1]
```diff
@@ -970,6 +970,12 @@ You can make an UPSERT with :code:`POST` and the :code:`Prefer: resolution=merge
 
 UPSERT operates based on the primary key columns, you must specify all of them. You can also choose to ignore the duplicates with :code:`Prefer: resolution=ignore-duplicates`.
 
+.. note::
```
Comment (steve-chavez) [inline api.rst]: We could be more precise about this, how about changing it to:

```
.. important::
  After creating a table or changing its primary key, you must refresh PostgREST schema cache for UPSERT to work properly. To learn how to refresh the cache see :ref:`schema_reloading`.
```

Make sure to add the `important`, that makes the note highlighted.

Comment (swuecho) [reply api.rst]: fixed.

**Comment by steve-chavez** [comment]
@swuecho Thank you! :smiley:

**Pull request merged**